OpenAI/Packages/com.openai.unity/Runtime/Realtime/CreateResponseRequest.cs

```diff
@@ -25,9 +25,9 @@ public CreateResponseRequest()
         /// <summary>
         /// Constructor.
         /// </summary>
-        /// <param name="options">Inference configuration <see cref="Realtime.Options"/> to override the <see cref="RealtimeSession.Options"/> for this response only.</param>
+        /// <param name="options">Inference configuration <see cref="Realtime.CreateResponseOptions"/> to override the <see cref="RealtimeSession.CreateResponseOptions"/> for this response only.</param>
         [Preserve]
-        public CreateResponseRequest(Options options)
+        public CreateResponseRequest(CreateResponseOptions options)
```

Comment:
Lets not change this since it'd be a breaking change. Also you can use the options for a one off request which is why I didn't name it this way here.

Reply:
In `CreateResponseRequest`, if you send the `Model` property, you get this error:

```
Error initializing session: System.Exception: [unknown_parameter] Unknown parameter: 'response.model'.
Type: invalid_request_error Parameter: response.model
```